src/Processor/Braintree.php

```diff
@@ -320,6 +340,52 @@ public function updateSubscription(Subscription $subscription): Subscription
         }
     }
 
+    private function changeSubscriptionBillingCycle(Subscription $subscription): Subscription
+    {
+        $customer = $subscription->getCustomer();
```

Comment:
This is pretty wordy, so if you can see a different way to go about it, please let me know. The wordiness has to do a bit with the static analysis as well as some of the nullable types. I have an idea to mitigate the null biz - but it can wait for another time 🙇